### DISCUSSION TITLE
I don't know what happened to it

### DISCUSSION THREAD

**Discussion by poki1584** [maruohon/litematica#273] [Q&A]
In begining I copy a normal build，but I change the computer and download it from the old computer，then it have be become like it.
![image](https://user-images.githubusercontent.com/76394685/102713788-8a5a4000-4305-11eb-92ef-fd498216b2f4.png)

**Comment by maruohon** [comment]
Set shaders to OFF in Optifine and restart the game.

I'm not sure why this started happening in 1.16 again, it was supposed to be fixed in 1.15.2...
It might get fixed once I work on the renderer rewrites at some point. But on the other hand the rendering works even with shaders enabled for some users (my own test instance included) even in the current version, which is the most confusing bit.
